app/datatables/datatable_base.rb

```diff
@@ -33,7 +33,10 @@ def paginate(objects)
   end
 
   def filter(objects)
-    if filter_params = params['filter']
+    if params['filter']
+      filter_params = params['filter'].reject{|name, value| special_filters.include? name}
```

Comment:
Not pushing a new commit for as its a tiny nitpick, but if you want you can use `_` in lieu of a variable name for variables you don't intend to use, that way if you run like a style checker or something it won't complain about declaring an unused variable.

Reply:
`filter_params` is used in the next if-clause.

Reply:
My bad, that was ambiguous, I meant `value` inside `reject` block.

Reply:
Doh. I'll remember that for the future.